generator/internal/stub_generator.cc

```diff
@@ -159,13 +160,14 @@ Status StubGenerator::GenerateHeader() {
     "\n"}},
                        // clang-format on
                        And(IsNonStreaming, Not(IsLongrunningOperation))),
-         MethodPattern({{R"""(
-  future<StatusOr<google::longrunning::Operation>> Async$method_name$(
+         MethodPattern(
+             {{R"""(  future<StatusOr<google::longrunning::Operation>> Async$method_name$(
       google::cloud::CompletionQueue& cq,
       std::unique_ptr<grpc::ClientContext> context,
       $request_type$ const& request) override;
+
```

Comment:
I was hoping we would settle on "code fragments never end with a blank line" so we could inject the blank line at the beginning of the code fragment (if needed), but as long as this is consistent I am okay.

Reply:
Yeah, I think that is what I was trying to get at with the "always using leading separators, except for the initial comment block" in the PR description.